### ISSUE TITLE
loss = -inf ,train cfair10 64*64*3

### ISSUE THREAD

**Issue by chl916185** [KangBK0120/D2GAN#1]
(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
64 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
65 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
66 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
67 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
68 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
69 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
70 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
71 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
72 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
73 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
74 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
75 epoch finished
-----------------------------------------------------------------

(200, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(400, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
(600, 'step')
tensor(-0., device='cuda:0') tensor(-inf., device='cuda:0')
76 epoch finished
-----------------------------------------------------------------
@KangBK0120

**Comment by KangBK0120** [comment]
Hi, thanks for replying

I'll check and update the codes ASAP with changing the version to pytorch 0.4.0

**Comment by SavinayShukla** [comment]
Hi, 

Is there any update to the -inf error ? Are the gradients exploding?
